### DISCUSSION TITLE
Not sure why my integration isn't getting any data from Manchester City Council...?

### DISCUSSION THREAD

**Discussion by jonxyz** [mampfes/hacs_waste_collection_schedule#755] [Q&A]
Hi there

Love the work you've put into this integration, but i'm having some trouble getting it working and i'd appreciate any help you can offer...! 🙂 

My config is very simple at the moment:

```
# Waste Collection Config
waste_collection_schedule:
    sources:
    - name: manchester_uk
      args:
        uprn: "xxxxxxxxxxx" 
```

I've redacted my 11 digit UPRN above for privacy reasons, but i've tried it with and without a leading 0, the manchester site html code has it with the leading 0 and the online UPRN search has it without the 0.

But in both cases i'm getting the following error in the logs, which implies no data being returned, but i'm not sure what i'm missing or where to go next?

```
Logger: waste_collection_schedule.source_shell
Source: custom_components/waste_collection_schedule/waste_collection_schedule/source_shell.py:136
Integration: waste_collection_schedule (documentation)
First occurred: 17:28:13 (1 occurrences)
Last logged: 17:28:13

fetch failed for source Manchester City Council: Traceback (most recent call last): File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source_shell.py", line 134, in fetch entries = self._source.fetch() File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source/manchester_uk.py", line 48, in fetch collection_type = img_tag["alt"] TypeError: 'NoneType' object is not subscriptable
```

**Comment by 5ila5** [comment]
Issue in #756
fixed by #758
